flink-core/src/main/java/org/apache/flink/api/common/JobExecutionResult.java

```diff
@@ -47,12 +48,23 @@ public JobExecutionResult(JobID jobID, long netRuntime, Map<String, Object> accu
 	 * Gets the net execution time of the job, i.e., the execution time in the parallel system,
 	 * without the pre-flight steps like the optimizer.
 	 *
-	 * @return The net execution time.
+	 * @return The net execution time in milliseconds.
 	 */
 	public long getNetRuntime() {
 		return this.netRuntime;
 	}
 
+    /**
+     * Gets the net execution time of the job, i.e., the execution time in the parallel system,
+     * without the pre-flight steps like the optimizer in a desired unit.
+     *
+     * @param desiredUnit the unit of the <tt>NetRuntime</tt>
+     * @return The net execution time in the desired unit.
+     */
+	public long elapsedNetRuntime(TimeUnit desiredUnit) {
```

Comment:
Hmm, good point. I was not aware of that.
@tillrohrmann Any particular reason why duration was changed from milliseconds to seconds?

IMO, it would make sense to the time with a higher resolution and convert it on need.